movie/src/components/body/Body.jsx

```diff
@@ -0,0 +1,14 @@
+import Leftbody from "../leftbody/Leftbody";
+import Rightbody from "../rightbody/Rightbody";
+import "./Body.css";
+
+const Body = () => {
```

Comment:
a more appropriate name would be `Container` or `MainContainer`

Reply:
> a more appropriate name would be `Container` or `MainContainer`

Yes, got it.